Makefile

```diff
@@ -65,6 +65,9 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=ocm-controller-manager-role crd webhook paths="./api/..." paths="./controllers/..." output:crd:artifacts:config=deploy/crds output:rbac:artifacts:config=deploy/templates
+	# Templatize the cluster role so that whomever is installing the chart can add labels.  See issue 518.
+	sed -i '/name: ocm-controller-manager-role/a\  labels: {{ $.Values.manager.clusterRole.labels | toJson }}' deploy/templates/role.yaml
```

Comment:
This is some kind of Post-operation thing that YOU need Dan if I understand correctly. I don't think this belongs in ocm-controller in general. If you need pre-installation adjustments to the role or something like that, maybe we can work around doing it with some kind of Helm value addition. I don't like templating the role in the Makefile itself. I don't think that's the right place to do it.